invisible-wallet/typescript/src/invisible_wallet_examples.ts

```diff
@@ -1,169 +1,197 @@
-import { ExecuteTransactionRequestType, SuiClient, SuiTransactionBlockResponse } from "@mysten/sui.js/client";
+// 1. Import everything we'll need for the rest of the tutorial
+import { 
+  WalletClient, 
+  KeyClient, 
+  createSuiClient, 
+  GasStationClient,
+  buildGaslessTransactionBytes, 
+} from "@shinami/clients";
 import { TransactionBlock } from "@mysten/sui.js/transactions";
-import { rpcClient } from "typed-rpc";
+import { verifyPersonalMessage } from '@mysten/sui.js/verify';
+
+// 2. Copy your access key value
+const ALL_SERVICES_TESTNET_ACCESS_KEY = "{{allServicesTestnetAccessKey}}";
+
+// // 3. Set up two wallet ids and a sercret for each
+const WALLET_ONE_ID = "{{walletOneId}}";
+const WALLET_ONE_SECRET = "{{walletOneSecret}}";
+const WALLET_TWO_ID = "{{walletTwoId}}";
+const WALLET_TWO_SECRET = "{{walletTwoSecret}}";
+
+// 4. Instantiate your Shinami clients
+const keyClient = new KeyClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const walletClient = new WalletClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const nodeClient = createSuiClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+
+// 5. Set up some variables we'll use in the examples below
+let sessionToken = "";
+let senderAddress = "";
+let senderSignature = null;
+
+// 6. Create the first wallet by generating a session token using its assocated secret
+sessionToken = await keyClient.createSession(WALLET_ONE_SECRET);
```

Comment:
I suggest promoting `ShinamiWalletSigner` to abstract away the session management like this:
https://github.com/shinamicorp/shinami-typescript-sdk/tree/main/packages/clients#invisible-wallet

Also I suggest us thinking through our strategy for maintaining sample code. We have at least these 3 places:
- SDK readmes / examples
- Tutorials
- This repo

It'll become more and more difficult for us to keep them in sync, and confusing for users to see different versions.

Reply:
I've updated my code to use  ShinamiWalletSigner. I'll test it, update the tutorial copy and code to reflect this, and then push my updates.

Also, good point about maintenance. I have a story in the backlog of the dev docs epic to pick up soon to do more planning around the next phase of the docs and have added this to it.